scripts/test-build-install-zip.sh

```diff
@@ -3,10 +3,16 @@
 set -euxo pipefail
 
 cleanup() {
-  r=$?
+  local r=$?
+  if [ "${r}" -ne 0 ]; then
+    # Ensure that the group where the failure happened is opened.
+    echo "^^^ +++"
+  fi
+  echo "~~~ elastic-package cleanup"
 
   # Dump stack logs
-  elastic-package stack dump -v --output build/elastic-stack-dump/build-zip
+  # Required containers could not be running, so ignore the error
+  elastic-package stack dump -v --output build/elastic-stack-dump/build-zip || true
```

Comment:
Example of the error to try to avoid:
https://buildkite.com/elastic/elastic-package/builds/5786#01983b8a-52f4-4906-a5fe-7379469778ed/350-608

This will help to continue running the other commands in the cleanup function.